http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -88,6 +89,14 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       _ <- F.raiseWhen(!hasPermission)(ForbiddenError(userInfo.userEmail))
 
       enableIntraNodeVisibility = req.labels.get(AOU_UI_LABEL).exists(x => x == "true")
+      // We only allow autopilot mode for members of CUSTOM_APP_USERS group
```

Comment:
Would it make sense to remove this constraints so you can test on Terra BEES? Just a thought

Reply:
I've tested a bunch on BEE already...this is so that this PR can be merged and we don't need to worry about it getting to prod